tests/rhwa/internal/dast/dast.go

```diff
@@ -0,0 +1,83 @@
+package dast
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift-kni/eco-gotests/tests/rhwa/internal/rhwaparams"
+	"gopkg.in/yaml.v3"
+)
+
+// PrepareRapidastConfig builds the trivy command and updates the rapidast configuration file.
+func PrepareRapidastConfig(destinationPath string) error {
+	workingDir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	templateDir := filepath.Join(workingDir, rhwaparams.RapidastTemplateFolder)
+	rapidastConfigFile := filepath.Join(templateDir, rhwaparams.RapidastTemplateFile)
+
+	file, err := os.Open(rapidastConfigFile)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	content, err := io.ReadAll(file)
+	if err != nil {
+		return err
+	}
+
+	yamlData := make(map[string]interface{})
+
+	err = yaml.Unmarshal(content, &yamlData)
+	if err != nil {
+		return err
+	}
+
+	sc, successful := yamlData["scanners"].(map[string]interface{})
+	if sc == nil || !successful {
+		return fmt.Errorf("yamlData is missing key \"scanners\"")
+	}
+
+	genericTrivy, successful := sc["generic_trivy"].(map[string]interface{})
+	if genericTrivy == nil || !successful {
+		return fmt.Errorf("yamlData is missing key \"generic_trivy\"")
+	}
+
+	genericTrivy["inline"] = fmt.Sprintf("trivy k8s --kubeconfig=/home/rapidast/.kube/config "+
```

Comment:
Is this `kubeconfig` the same defined in `KUBECONFIG` env variable read in `far.go:79`? Or, should this be the same value?

Reply:
Although both are pointing to the same kubeconfig file, this is the location for the `kubeconfig` inside the container, where the rapidast software expects it. In `far.go` it is referenced as a mounted volume for the podman command.

Reply:
since you have the path it, I would consider passing it as argument then

Reply:
ah, nevermind. I saw now that you mount the location of the kubeconfig in podman

Reply:
Exactly, and that location in the podman command is meant to be hardcoded, not parametrized.